core/src/primary_keys.rs

```diff
@@ -9,41 +9,47 @@ lazy_static! {
             "".to_string(),
             "TABLE_CAT".to_string(),
             BsonTypeInfo::STRING,
+            TypeMode::Standard,
             Nullability::NULLABLE
         ),
         MongoColMetadata::new_metadata_from_bson_type_info(
             "",
             "".to_string(),
             "TABLE_SCHEM".to_string(),
             BsonTypeInfo::STRING,
+            TypeMode::Standard,
             Nullability::NULLABLE
         ),
         MongoColMetadata::new_metadata_from_bson_type_info(
             "",
             "".to_string(),
             "TABLE_NAME".to_string(),
             BsonTypeInfo::STRING,
+            TypeMode::Standard,
             Nullability::NO_NULLS
         ),
         MongoColMetadata::new_metadata_from_bson_type_info(
             "",
             "".to_string(),
             "COLUMN_NAME".to_string(),
             BsonTypeInfo::STRING,
+            TypeMode::Standard,
             Nullability::NO_NULLS
         ),
         MongoColMetadata::new_metadata_from_bson_type_info(
             "",
             "".to_string(),
             "KEY_SEQ".to_string(),
             BsonTypeInfo::INT,
+            TypeMode::Standard,
             Nullability::NO_NULLS
```

Comment:
There is no difference between the simple and regular versions of `BsonTypeInfo::INT` and `BsonTypeInfo::STRING`, so we can just set this to `TypeMode::Standard`.